event-sourcing/event-sourcing-core/src/test/java/org/apache/james/eventsourcing/EventSourcingSystemTest.java

```diff
@@ -43,7 +43,7 @@
 
     String PAYLOAD_1 = "payload1";
     String PAYLOAD_2 = "payload2";
-    TestAggregateId AGGREGATE_ID = TestAggregateId.testId(42);
+    TestAggregateId AGGREGATE_ID = TestAggregateId.apply(42);
```

Comment:
euurk?

Reply:
rewritten in scala in a later commit

Reply:
it's still here: https://github.com/linagora/james-project/pull/2972/files#diff-d7be851b7817d00bb97f8e3db252980cR36

Reply:
fixed :)